app-conf/GeneralConf.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright 2016 LinkedIn Corp.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<!-- General Configurations -->
+
+<configuration>
+  <property>
+    <name>drelephant.analysis.completed.thread.count</name>
+    <value>5</value>
+    <description>Number of threads to analyse completed(succeeded/failed) jobs.</description>
+  </property>
+  <property>
+    <name>drelephant.analysis.realtime.thread.count</name>
```

Comment:
We need to find optimal combination of drelephant.analysis.realtime.thread.count and drelephant.analysis.fetch.interval to capture all the current running jobs.